src/neo/Ledger/Blockchain.cs

```diff
@@ -65,6 +65,7 @@ public class RelayResult { public IInventory Inventory; public VerifyResult Resu
         private readonly Dictionary<UInt256, Block> block_cache = new Dictionary<UInt256, Block>();
         private readonly Dictionary<uint, LinkedList<Block>> block_cache_unverified = new Dictionary<uint, LinkedList<Block>>();
         internal readonly RelayCache ConsensusRelayCache = new RelayCache(100);
+        internal readonly RelayCache OracleRelayCache = new RelayCache(100);
```

Comment:
Can they be merged?

Reply:
I think so, we can use the same relay cache with double capacity.

Reply:
Please check https://github.com/neo-project/neo/pull/1631